### PR TITLE
Migrate command `triggeraudit`

### PR DESCRIPTION
**Pull request by zymap**
Descriptions of the changes in this PR:

- Using `bkctl` run command `triggeraudit`

### Motivation

#2011

### CODE REVIEW

**Comment by zymap** [comment]
run bookkeeper-server bookie tests
run bookkeeper-server replication tests

File: tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/ToggleCommandGroup.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import org.apache.bookkeeper.tools.cli.commands.toggle.TriggerAuditCommand;
+import org.apache.bookkeeper.tools.common.BKFlags;
+import org.apache.bookkeeper.tools.framework.CliCommandGroup;
+import org.apache.bookkeeper.tools.framework.CliSpec;
+
+/**
+ * Commands on some specific operation.
+ */
+public class ToggleCommandGroup extends CliCommandGroup<BKFlags> {
```
Comment (sijie) [inline tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/ToggleCommandGroup.java]: command group - 'autorecovery' ?

**Comment by zymap** [comment]
run integration tests

**Comment by zymap** [comment]
run bookkeeper-server bookie tests

**Comment by zymap** [comment]
run bookkeeper-server bookie tests

**Comment by zymap** [comment]
run bookkeeper-server remaining tests

**Comment by zymap** [comment]
run bookkeeper-server remaining tests

**Pull request merged**